site/content/docs/api-checks/_index.md

```diff
@@ -13,64 +13,107 @@ aliases:
 You can assert on different sources. These could be:
 - The HTTP status code returned from the API
 - Something missing or required within the response body
 - A specific response header
 - A specific response time
 
 [Read more about assertions](assertions)
-![assertions](/docs/images/api-checks/overview-assertions.png)
 
-**6. Locations**
+### Scheduling & locations
+
+You can configure your checks to run from our [public](/docs/monitoring/global-locations/) locations, or use a Checkly Agent to host your own [private](/docs/private-locations/) locations. If you don't select more than one location and you've disabled retrying checks from the same location, we will pick a random location when retrying checks.
+
+Checkly runs your API checks based on an interval you set. The shortest interval you can run is every 10 seconds and the longest is every 24 hours.
+
+### Retries & alerting
+
+Select your preferred [retry strategy](/docs/alerting-and-retries/retries/) for failed checks.
+
+Choose which [alert channels](/docs/alerting-and-retries/alert-channels/) to get notified through when your check runs into issues. If we don't have your preferred alert method, why not try out our [webhooks](/docs/alerting-and-retries/webhooks/)?
+
+### Testing
+
+You can run your check as an [E2E test](/docs/testing) locally or from your CI/CD pipeline to validate your freshly deployed application. Use the Checkly CLI, or configure integrations with Vercel and GitHub.
+
+## CLI example
+
+The [Checkly CLI](/guides/getting-started-with-monitoring-as-code/) gives you a JavaScript/TypeScript-native workflow for coding, testing and deploying synthetic monitoring at scale, from your code base.
+
+You can define an API check via the CLI. For example:
 
-You can configure your checks to run from our ever-growing amount of global [public](/docs/monitoring/global-locations/) locations or use a Checkly Agent to host your own [private](/docs/private-locations/)
-If you don't select more than one data center location, we will pick a random location when retrying checks if you have "double check" enabled.
-![locations](/docs/images/api-checks/overview-locations.png)
+```ts {title="hello-api.check.ts"}
+import { ApiCheck, AssertionBuilder, Frequency } from 'checkly/constructs'
 
-**7. Scheduling**
+new ApiCheck('hello-api-1', {
+  name: 'Hello API',
+  activated: true,
+  frequency: Frequency.EVERY_1M,
+  request: {
+    method: 'GET',
+    url: 'https://mac-demo-repo.vercel.app/api/hello',
+    assertions: [
+      AssertionBuilder.statusCode().equals(200)
+    ],
+  }
+})
+```
 
-Checkly runs your API checks based on an interval you select. We have a minimum and maximum interval for your API checks.
-The quickest schedule you can run is every 10 seconds and the slowest is every 24 hours.
-![schedule](/docs/images/api-checks/overview-schedule.png)
+The above example defines:
+- The basic check properties like `name`, `activated` etc.
+- The HTTP method `GET` the `url`.
```

Comment:
@guolau this is phrased weirdly. Perhaps "The HTTP method `GET` and the target `url` of the request?